bin/node/cli/Cargo.toml

```diff
@@ -72,6 +72,7 @@ pallet-transaction-payment = { version = "2.0.0", path = "../../../frame/transac
 frame-support = { version = "2.0.0", default-features = false, path = "../../../frame/support" }
 pallet-im-online = { version = "2.0.0", default-features = false, path = "../../../frame/im-online" }
 pallet-authority-discovery = { version = "2.0.0",  path = "../../../frame/authority-discovery" }
+pallet-society = { version = "2.0.0",  path = "../../../frame/society" }
```

Comment:
Why do we need this dependency?

Reply:
It gets used here: https://github.com/paritytech/substrate/pull/4651/files#diff-c9f62bd741e70b8157fa630030c33a57R298